client/src/Components/StudentGroup.js

```diff
@@ -0,0 +1,74 @@
+import React, {Component} from 'react';
+import css from './Student.module.css'
+
+class StudentGroup extends Component{
+
+    constructor(props)
+    {
+        super(props)
+        this.state={
+            leavegroupbtn: true
```

Comment:
`leavegroupbtn` does not really describe what this is doing. Is there a better name that you can use for this key?